src/js/components/DataFilters/DataFilters.js

```diff
@@ -86,7 +86,7 @@ export const DataFilters = ({ drop, children, heading, layer, ...rest }) => {
   const content = (
     <DataForm
       pad={controlled ? 'medium' : undefined}
-      gap="small"
+      gap="xsmall"
```

Comment:
HPE FormFields already have spacing built into FormField styling, so `gap` shouldn't be used at all. Wondering if this value should be removed or theme driven?

Reply:
Good catch. I've made a few layout adjustments to address.